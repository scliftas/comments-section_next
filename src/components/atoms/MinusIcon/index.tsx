import type { FC } from "react";

type Props = {
  className?: string;
  title: string;
};

const MinusIcon: FC<Props> = (props) => {
  const { className = "", title } = props;

  return (
    <svg
      className={className}
      width="11"
      height="11"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title>{title}</title>
      <path d="M9.256 2.66c.204 0 .38-.056.53-.167.148-.11.222-.243.222-.396V.722c0-.152-.074-.284-.223-.395a.859.859 0 0 0-.53-.167H.76a.859.859 0 0 0-.53.167C.083.437.009.57.009.722v1.375c0 .153.074.285.223.396a.859.859 0 0 0 .53.167h8.495Z" />
    </svg>
  );
};

export default MinusIcon;
