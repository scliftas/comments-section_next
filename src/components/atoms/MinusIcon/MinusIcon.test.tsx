import { render } from "@testing-library/react";
import MinusIcon from ".";

describe("Minus Icon component", () => {
  it("matches the snapshot", () => {
    const { container } = render(
      <MinusIcon className="fill-black" title="Minus icon test title" />
    );

    expect(container).toMatchSnapshot();
  });
});
