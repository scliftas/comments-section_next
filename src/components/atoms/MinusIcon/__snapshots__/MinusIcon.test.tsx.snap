// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Minus Icon component matches the snapshot 1`] = `
<div>
  <svg
    class="fill-black"
    height="11"
    width="11"
    xmlns="http://www.w3.org/2000/svg"
  >
    <title>
      Minus icon test title
    </title>
    <path
      d="M9.256 2.66c.204 0 .38-.056.53-.167.148-.11.222-.243.222-.396V.722c0-.152-.074-.284-.223-.395a.859.859 0 0 0-.53-.167H.76a.859.859 0 0 0-.53.167C.083.437.009.57.009.722v1.375c0 .153.074.285.223.396a.859.859 0 0 0 .53.167h8.495Z"
    />
  </svg>
</div>
`;
